src/react-app/src/configs/i18n.ts

```diff
@@ -0,0 +1,23 @@
+import i18n from 'i18next';
+import Backend from 'i18next-http-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { initReactI18next } from 'react-i18next';
+
+const backendOptions = {
+  loadPath: `${process.env.PUBLIC_URL}/i18n/{{lng}}.json`,
+};
+
+i18n
+  .use(Backend)
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    backend: backendOptions,
+    fallbackLng: 'en-US',
+    debug: true,
```

Comment:
It looks like your `lowerCaseLng: true,` from your [other branch](https://github.com/MyPureCloud/genesys-cloud-client-auth/compare/PCM-1645-Translations) never made it over.

Reply:
Whoops. Will need to change `en-US.json` to `en-us.json` too.